packages/jsii-reflect/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "jsii-reflect",
+  "version": "0.7.11",
+  "description": "strongly-typed reflection library and tools for jsii",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "bin": {
+    "jsii-tree": "bin/jsii-tree"
+  },
+  "scripts": {
+    "build": "tsc",
+    "watch": "tsc -w",
+    "test": "jest",
+    "package": "package-js"
+  },
+  "dependencies": {
+    "fs-extra": "^7.0.1",
+    "jsii-spec": "^0.7.11",
+    "yargs": "^12.0.5",
+    "colors": "^1.3.2",
```

Comment:
Order :)

Reply:
I'll pass. I don't see tons of value in maintaining alphabetical order in these lists. It's more work than benefit IMHO